blue_green_deploy.go

```diff
@@ -30,6 +31,12 @@ type BlueGreenDeploy struct {
 	ErrorFunc  ErrorHandler
 }
 
+type AppScaleParameters struct {
```

Comment:
Would it be better if this was a substructure of the BlueGreenDeploy struct? Then we wouldn't have to pass it around all the time in function calls.